<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectSave</class>
 <widget class="QDialog" name="ProjectSave">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1195</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save Project</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="3">
      <widget class="QPushButton" name="btnBrowseFilePath">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Project Location</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Workspaces</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" rowspan="3">
      <widget class="QTreeWidget" name="workspaceList">
       <column>
        <property name="text">
         <string>Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Size</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>ws1</string>
        </property>
        <property name="checkState">
         <enum>Checked</enum>
        </property>
        <property name="text">
         <string>10mb</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QPushButton" name="btnSaveAll">
       <property name="text">
        <string>Save All</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QLineEdit" name="projectPath"/>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="3">
      <widget class="QTreeWidget" name="excludedWindows">
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="3" column="1" colspan="3">
      <widget class="QTreeWidget" name="includedWindows">
       <column>
        <property name="text">
         <string>Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Type</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Size</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Excluded Windows</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Included Windows</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
